src/pages/home/report/ReportActionCompose/SuggestionMention.js

```diff
@@ -233,8 +235,15 @@ function SuggestionMention({
     );
 
     useEffect(() => {
+        if (value.length < previousValue.length) {
+            // A workaround to not show the suggestions list when the user deletes a character before the mention.
+            // It is caused by a buggy behavior of the TextInput on iOS. Should be fixed after migration to the Fabric Arc.
+            // See: https://github.com/facebook/react-native/pull/36930#issuecomment-1593028467
+            return;
+        }
+
         calculateMentionSuggestion(selection.end);
-    }, [selection, calculateMentionSuggestion]);
+    }, [selection, value, previousValue, calculateMentionSuggestion]);
```

Comment:
Both `value` and `previousValue` are necessary, right?

Reply:
I would omit both values from dependencies, but in that case, we get a lint error – it requires both to be present

Reply:
Oh, sure. The truth is that we opt-out quite often from that lint... In this case I would go whichever path is less likely to create another regression...

Reply:
I agree with you @cubuspl42, we should follow the path that is less likely to create another regression.